astroplan/tests/test_sites.py

```diff
@@ -33,6 +34,12 @@ def test_add_site():
     retrieved_location = get_site(new_site_name)
     assert retrieved_location == new_site_location
 
+def test_Observer_classmethod():
+    site_name = 'kpno'
+    kpno = Observer.at_site(site_name)
+    assert kpno.location == get_site(site_name)
+    assert kpno.name == site_name
```

Comment:
you should also test that the ``kwargs`` usage passes through like it's supposed to.